mmv1/products/containerattached/api.yaml

```diff
@@ -0,0 +1,273 @@
+        output: true
+      - !ruby/object:Api::Type::String
+        name: description
+        description: |
+          A human readable description of this attached cluster. Cannot be longer
+          than 255 UTF-8 encoded bytes.
+      - !ruby/object:Api::Type::String
+        name: distribution
+        description: |
+          The Kubernetes distribution of the underlying attached cluster. Supported values:
+          "eks", "aks".
+        required: true
+        input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: errors
+        description: |
+          A set of errors found in the cluster.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: message
+            description: |
+              Human-friendly description of the error.
+            item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: fleet
+        description: |
+          Fleet configuration.
+        required: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: membership
+            description: |
+              The name of the managed Hub Membership resource associated to this
+              cluster. Membership names are formatted as
+              projects/<project-number>/locations/global/membership/<cluster-id>.
+            output: true
+          - !ruby/object:Api::Type::String
+            name: project
+            description: |
+              The number of the Fleet host project where this cluster will be registered.
+            required: true
+            input: true
+      - !ruby/object:Api::Type::String
+        name: kubernetesVersion
+        description: |
+          The Kubernetes version of the cluster.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: location
+        description: |
+          The location for the resource
+        input: true
+        required: true
+        url_param_only: true
+      - !ruby/object:Api::Type::NestedObject
+        name: loggingConfig
+        description: |
+          Logging configuration.
+        send_empty_value: true
+        allow_empty_object: true
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: componentConfig
+            description: |
+              The configuration of the logging components
+            send_empty_value: true
+            allow_empty_object: true
+            properties:
+            - !ruby/object:Api::Type::Array
+              name: enableComponents
+              description: |
+                The components to be enabled.
+              send_empty_value: true
+              allow_empty_object: true
+              item_type: !ruby/object:Api::Type::Enum
+                name: 'component'
+                description: |
+                  The components of the logging configuration.
+                values:
+                  - :COMPONENT_UNSPECIFIED
```

Comment:
Generally I prefer not to include UNSPECIFIED enum values, since they generally do nothing compared to not providing a value at all, and sometimes even cause errors. Just wanted to double-check that this is a meaningful value here?

Reply:
It is not meaningful. I have removed it.